projects/storefrontapp/src/app/spartacus/features/administration-feature.module.ts

```diff
@@ -10,7 +10,7 @@ import { AdministrationRootModule } from '@spartacus/organization/administration
   declarations: [],
   imports: [AdministrationRootModule],
   providers: [
-    provideConfig({
+    provideConfig(<CmsConfig>{
```

Comment:
Shouldn't we still in this case i18n config from here? Applies to more features in our storefrontapp

Reply:
Thanks, updated.